### DISCUSSION TITLE
Revert to 1.2 UI - even possible?

### DISCUSSION THREAD

**Discussion by kainah** [helge17/tuxguitar#26] [General]
Hi guys,

Since I lack any knowledge of Java, have to ask around here. It's a small issue but still.
Is there any way to modify the all-new user interface with the sidebar (since 1.3, I guess) to look like 1.2 one with the horizonal rows of buttons instead of a sidebar? Or is the new UI just hard-coded that way?
I'm just so used to the old UI, having spent more than a decade using it.
If changing the UI would be too hard, then no big deal.

**Comment by guiv42** [comment]
hi @kainah. Late answer, but still better than nothing.
It's always very difficult to revert someone else's decision... And when considering user interface, there is a chance that some users complain if we decide to do so. The best solution would probably to make this configurable, but it would not be so simple as everything is pretty much hard-coded.
Even worse: several skins have their icons in the main toolbar adapted to this layout, with buttons to show/hide edit toolbar and tracks table clearly showing this geographical layout (edit toolbar on the left / tracks table on the bottom):
![breeze](https://github.com/helge17/tuxguitar/assets/129443524/76a24f2b-dad6-43ef-ba92-e2579cece101) ![symbolic](https://github.com/helge17/tuxguitar/assets/129443524/0cca4f62-dee5-447f-b77f-0cb88658bf8a) ![oxygen](https://github.com/helge17/tuxguitar/assets/129443524/1e77c1cd-0082-4929-855e-3e8f9d23634a)

So I think it's better to keep this layout unchanged.

**Reply by kainah** [reply]
@guiv42 Yeah, I guess so. Besides, that's the UI scheme they now follow at Arobas for Guitar Pro since GP6. But old habits die hard, I'm still on TG 1.2 even on win11. Anyway, thanks mate.

**Comment by guiv42** [comment]
Hi @kainah 
If you're still around, you might like the latest pre-releases: the mail toolbar is now fully configurable. You should be able to recreate a 1.2-like environment.
Regards

**Reply by kainah** [reply]
Hi! Awesome news! Thanks for remembering me, mate! Gotta test it out. Eternal gratitude goes out to y'all.
